.github/workflows/test.yml

```diff
@@ -9,6 +9,20 @@ on:
       - '.gitignore'
 
 jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install dependencies
```

Comment:
🔧 Ensure that the 'requirements.txt' file is up-to-date with all necessary dependencies for the tests.